Tests/playwright.config.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
```

Comment:
Does this file have to be checked in?  Isn't it generated in the build process?

Reply:
resolved